utils/web3/unreal.ts

```
@@ -0,0 +1,149 @@
+    txHash = await wallet.writeContract({
+      address: token,
+      abi: unrealTokenAbi,
+      functionName: "transferFrom",
+      args: [from, to, amount],
+      account: wallet.account,
+      chain: wallet.chain,
+      // Add high priority fee to ensure transaction is processed quickly
+      // maxPriorityFeePerGas: 2_000_000_000n, // 2 Gwei
+      // maxFeePerGas: 100_000_000_000n, // 100 Gwei max fee
+    })
+  } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)